src/Tribe/Views/V2/View.php

```diff
@@ -1417,7 +1417,7 @@ protected function setup_template_vars() {
 			'after_events'           => tribe( Advanced_Display::class )->get_after_events_html( $this ),
 			'display_events_bar'     => $this->filter_display_events_bar( $this->display_events_bar ),
 			'disable_event_search'   => tribe_is_truthy( tribe_get_option( 'tribeDisableTribeBar', false ) ),
-			'live_refresh'           => tribe_is_truthy( tribe_get_option( 'liveFiltersUpdate', true ) ),
+			'live_refresh'           => tribe_is_truthy( 'automatic' === tribe_get_option( 'liveFiltersUpdate', 'automatic' ) ),
```

Comment:
:warning: Array double arrow not aligned correctly; expected 9 space(s) between "'live_refresh'" and double arrow, but found 11.

<sub>hash: 359a913c3e84786981e853e052d12c72</sub>